app/services/places/visits/create.rb

```
@@ -0,0 +1,89 @@
+      if user.safe_settings.distance_unit == :km
+        DEFAULT_PLACE_RADIUS / ::DISTANCE_UNITS[:km]
+      else
+        DEFAULT_PLACE_RADIUS / ::DISTANCE_UNITS[user.safe_settings.distance_unit.to_sym]
+      end
+
+    points = Point.where(user_id: user.id)
+                  .near([place.latitude, place.longitude], place_radius, user.safe_settings.distance_unit)
+                  .order(timestamp: :asc)
+
+    points.group_by { |point| Time.zone.at(point.timestamp).strftime('%Y-%m') }
+  end
```

RuboCop (1.81.7)

[convention] Assignment Branch Condition size for `place_points` is too high. [<3, 26, 4> 26.48/17]

(Metrics/AbcSize)